.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+      - "*-stable"
+  pull_request:
+    branches:
+      - master
+      - "*-stable"
```

Comment:
Would you mind adding `main` to the list of branches here? While there aren't any plans yet to wholesale move repos away from using `master` as the default branch, adding `main` here will make that task easier.